lib/pg_repack.sql.in

```diff
@@ -38,21 +40,25 @@ AS $$
             from '\d+\.\d+(?:\.\d+)?'), '.')::int[]
     ) as x (tokens);
 $$
-LANGUAGE SQL IMMUTABLE;
+LANGUAGE SQL IMMUTABLE
+SET search_path to 'pg_catalog';
```

Comment:
I don't think those functions can be called from a user-initiated sessions? In case they can it'd be better to `SET search_path to 'pg_catalog, pg_temp';` to specify the precedence, see https://www.postgresql.org/docs/current/static/sql-createfunction.html#SQL-CREATEFUNCTION-SECURITY (it was initially written w.r.t. security-definer functions, but looks like making sense for ordinary ones as well if we need to trust them what they're doing).

Reply:
Looking better at things, the session repack uses has already a restrictive search path

https://github.com/reorg/pg_repack/blob/900bb8369bd3e28d14e67f017325e1109a12418c/bin/pg_repack.c#L548-L549

So I don't think this is necessary at all.